op-node/rollup/derive/l1_block_info.go

```diff
@@ -54,6 +61,9 @@ type L1BlockInfo struct {
 	BlobBaseFee       *big.Int // added by Ecotone upgrade
 	BaseFeeScalar     uint32   // added by Ecotone upgrade
 	BlobBaseFeeScalar uint32   // added by Ecotone upgrade
+
+	InteropSetSize uint8      // added by Interop upgrade
```

Comment:
We don't need the size in this struct but we can make a getter that wraps `len(dependencySet)`